package.json

```diff
@@ -76,7 +76,7 @@
   },
   "devDependencies": {
     "@codemirror/lang-javascript": "^6.0.2",
-    "@napi-rs/cli": "^2.13.2",
+    "@napi-rs/cli": "3.0.0-alpha.5",
```

Comment:
I *think* I can avoid this though. You want me to try?

Reply:
I did rollback to the current CLI tool version and it works 👍 

In the current CLI tool, support for zig is a bit more experimental (the new one relies on `cargo-zigbuild`, which is more reliable), so I had to rollback to zig 0.9.1 because I couldn't get it to work with 0.10.1

Reply:
:pray: